resolwe/test/testcases/process.py

```diff
@@ -213,7 +213,7 @@ def setUp(self):
         self._profiler = TestProfiler(self)
         self._preparation_stage = 0
         self._executed_processes = set()
-        self._keep_data = False
+        self._keep_data = settings.FLOW_MANAGER_TEST_KEEP_DATA
```

Comment:
This attribute is already set in this class's `TestCaseHelpers` mixin so it would best to remove this line altogether.